testdata/goldens/dotnet/Google.Cloud.Asset.V1.UpdateFeedRequest.html

```diff
@@ -8,13 +8,13 @@
     {% verbatim %}
     <div>
       <article data-uid="Google.Cloud.Asset.V1.UpdateFeedRequest">
-<h1 class="page-title">Class UpdateFeedRequest</h1>
+<h1 class="page-title">Cloud Asset V1 API - Class UpdateFeedRequest</h1>
   
   
   <div class="codewrapper">
     <pre class="prettyprint"><code>public sealed class UpdateFeedRequest : IMessage&lt;UpdateFeedRequest&gt;, IEquatable&lt;UpdateFeedRequest&gt;, IDeepCloneable&lt;UpdateFeedRequest&gt;, IBufferMessage, IMessage</code></pre>
   </div>
-  <div class="markdown level0 summary"><p>Update asset feed request.</p>
+  <div class="markdown level0 summary"><p>Reference documentation and code samples for the Cloud Asset V1 API class UpdateFeedRequest.</p><p>Update asset feed request.</p>
```

Comment:
Is there a way to capitalize the types? like
`Reference documentation and code samples for the Cloud Asset V1 API Class UpdateFeedRequest.`
for `Class` instead of `class`.

Reply:
Sure, I think that would look better. Right now we take the lowercase of the type ([Code](https://github.com/googleapis/doc-templates/blob/main/third_party/docfx/templates/devsite/common.js#L124)), but we can change this leaving casing as is. I'll do this as a follow-up.

Reply:
SGTM. Could you file an issue for it?

Reply:
Done, #416 !